### PR TITLE
[DEV-1927] add wallet apiKey

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes
- [ ] **Update database/environment**

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Comment by Yannick1712** [comment]
Add migration

File: src/subdomains/generic/user/models/kyc/kyc.service.ts [hunk 1]
```diff
@@ -127,13 +127,11 @@ export class KycService {
     const dfxUser = await this.userRepo.findOne({ where: { id: userId }, relations: ['userData'] });
     if (!dfxUser) throw new NotFoundException('DFX user not found');
     if (!KycCompleted(dfxUser.userData.kycStatus)) throw new ConflictException('KYC required');
-
-    const apiKey = this.walletService.getApiKeyInternal(wallet.name);
-    if (!apiKey) throw new Error(`ApiKey for wallet ${wallet.name} not available`);
+    if (!wallet.apiKey) throw new Error(`ApiKey for wallet ${wallet.name} not available`);
```
Comment (davidleomay) [inline src/subdomains/generic/user/models/kyc/kyc.service.ts:130]: WalletService can be removed from constructor params?

File: src/subdomains/generic/user/services/webhook/webhook.service.ts [hunk 2]
```diff
@@ -74,9 +74,7 @@ export class WebhookService {
   ): Promise<void> {
     try {
       if (!user.wallet.isKycClient || !user.wallet.apiUrl) return;
-
-      const apiKey = this.walletService.getApiKeyInternal(user.wallet.name);
-      if (!apiKey) throw new Error(`ApiKey for wallet ${user.wallet.name} not available`);
+      if (!user.wallet.apiKey) throw new Error(`ApiKey for wallet ${user.wallet.name} not available`);
```
Comment (davidleomay) [inline src/subdomains/generic/user/services/webhook/webhook.service.ts:77]: Same here?

**Review by davidleomay — APPROVED** [review]

**Review by davidleomay — CHANGES_REQUESTED** [review]
Update environment (Bicep, parameter files)

**Review by davidleomay — APPROVED** [review]

**Pull request merged**